erpnext/accounts/doctype/sales_invoice/sales_invoice.py

```diff
@@ -199,6 +200,16 @@ def on_submit(self):
 		if "Healthcare" in active_domains:
 			manage_invoice_submit_cancel(self, "on_submit")
 
+	def validate_pos_return(self):
+
+		if self.is_pos and self.is_return:
+			total_amount_in_payments = 0
+			for payment in self.payments:
+				total_amount_in_payments = payment.amount
+
+			if -total_amount_in_payments > -self.rounded_total:
```

Comment:
Better way of writing this condition would be `total_amount_in_payments < self.rounded_total`